modules/visual_script/visual_script_property_selector.cpp

```diff
@@ -469,10 +469,11 @@ void VisualScriptPropertySelector::_item_selected() {
 
 		at_class = ClassDB::get_parent_class_nocheck(at_class);
 	}
-	Map<String, DocData::ClassDoc>::Element *T = dd->class_list.find(class_type);
+	Vector<String> functions = name.rsplit("/", false);
+	at_class = functions.size() > 3 ? (class_type == functions[functions.size() - 2] ? class_type : functions[functions.size() - 2]) : class_type;
```

Comment:
That's a bit hard to read with two ternary operators... but also seems unnecessary? Whether `functions[functions.size() - 2] == class_type` or not you will get the value you need with `functions[functions.size() - 2]`.

```cpp
at_class = functions.size() > 3 ? functions[functions.size() - 2] : class_type;
```

Is it intended to check for `> 3` (i.e. min 4 items) and not `> 2`?

Reply:
@akien-mga You're absolutely right. It's redundant, I just noticed. My bad! I'll fix it right away...